### DISCUSSION TITLE
Cannot connect to Client

### DISCUSSION THREAD

**Discussion by winotoj** [rustdesk/rustdesk#8511] [Q&A]
I just installed free version on DMZ, when both client on LAN, I could remote (correctly if I'm wrong, I think I must use /r at the end of the ID), but from my LAN i couldn't connect to my laptop at home (both windows 11), it's timing out, both said Ready so i think both of them could connect to server, any other setup that I need on Fortigate?
Configurations:
Server behind fortigate with virtual IP, allowed TCP/UDP 21114-21119 both way and to/from LAN
For testing, PC at the office allow any/any, RustDesk Service is OFF (it was OK when both are on LAN, even when i removed the service)
There is no rule from WAN to LAN since I want to connect to outside.
Laptop Windows firewall is OFF, RustDesk service is ON.

I saw https://community.fortinet.com/t5/FortiGate/Technical-Tip-Cannot-use-RustDesk-from-outside-to-connect/ta-p/274423 but this one is from outside to inside.

Second issue: when I installed using powershell: rustdesk-version-host=[External IP],key=[key].exe --silent-install, it always ignores host and key until I remove and reinstall, is this normal?

**Comment by rustdesk** [comment]
Usually nobody will help you here, try out https://discord.com/invite/nDceKgxnkV
